services/ride-service/src/main/java/com/Harevich/ride_service/util/mapper/PageMapper.java

```diff
@@ -0,0 +1,25 @@
+package com.Harevich.ride_service.util.mapper;
+
+
+import com.Harevich.ride_service.dto.response.PageableResponse;
+import org.mapstruct.InjectionStrategy;
+import org.mapstruct.Mapper;
+import org.springframework.data.domain.Page;
+
+@Mapper(
+        componentModel = "spring",
+        injectionStrategy = InjectionStrategy.CONSTRUCTOR
+)
+public interface PageMapper{
+
```

Comment:
По возможности вместо строк/чисел следует пользоваться константами, желательно уже созданными, если они есть.
```suggestion
@Mapper(
        componentModel = MappingConstants.ComponentModel.SPRING,
        injectionStrategy = InjectionStrategy.CONSTRUCTOR
)
public interface PageMapper {

```